app/pages/DomainManager/ClaimNameForPayment.js

```diff
@@ -56,7 +56,6 @@ export default class ClaimNameForPayment extends Component {
       this.setState({
         isLoading: true,
       });
-      await this.props.waitForPassphrase();
```

Comment:
safe to remove here since passphrase is enforced once already here: https://github.com/kyokan/bob-wallet/blob/master/app/ducks/names.js#L325